### PR TITLE
vod.tvp.pl

### PR DESCRIPTION
**Pull request by FadeMind**
https://vod.tvp.pl/

### CODE REVIEW

**Comment by Gusted** [comment]
Hmm, the black text doesn't seems to occur for me... 
image

**Comment by FadeMind** [comment]
@Gusted look without fixes

Edge Chromium stable latest
Dark Reader Dev Build latest
Setup: Dynamic Filter




with Filter/Filter+ looking fine.

**Comment by Gusted** [comment]
Seems like a chromium-only bug? Il check it out what's going on here.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -12471,6 +12471,19 @@ INVERT
 
 ================================
 
+vod.tvp.pl
+
+INVERT
+img[alt="serwisy tvp"]
+
+CSS
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:12479]: You can remove this, stumbled upon a bug and should be fixed within the code. Anyway, thanks for fixing it and accidentally stumbling upon a bug 😄

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:12479]: you meant remove CSS section only and save INVERT image?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:12479]: Yep!

**Comment by FadeMind** [comment]
@Gusted ready to go.

**Review by Gusted — APPROVED** [review]
Once again thanks for stumbling upon the bug 😃

**Pull request merged**